packages/app/src/app/hooks/useSubscription.ts

```diff
@@ -32,11 +38,49 @@ export const useSubscription = () => {
 
   const isEligibleForTrial = isTeamSpace && !hasPastOrActiveSubscription;
 
+  const numberOfEditors = isTeamSpace
+    ? activeTeamInfo?.userAuthorizations?.filter(
+        ({ authorization }) =>
+          authorization === TeamMemberAuthorization.Admin ||
+          authorization === TeamMemberAuthorization.Write
+      ).length
+    : 1; // Personal
+
+  const numberOfSeats = activeTeamInfo?.subscription?.quantity || 1;
+
+  const hasMaxNumberOfEditors =
+    !hasActiveSubscription &&
+    numberOfEditors === activeTeamInfo?.limits.maxEditors;
+
+  const numberOfEditorsIsOverTheLimit =
+    !hasActiveSubscription &&
+    numberOfEditors > activeTeamInfo?.limits.maxEditors;
```

Comment:
To fix the type errors
```
src/app/hooks/useSubscription.ts(57,5): error TS2532: Object is possibly 'undefined'.
src/app/hooks/useSubscription.ts(57,23): error TS2533: Object is possibly 'null' or 'undefined'.
```
```suggestion
    numberOfEditors && numberOfEditors > activeTeamInfo?.limits?.maxEditors;
```

Reply:
Still not sure why they don't show up in the editor, probably something with not picking up custom-named deeply nested tsconfigs.